dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -427,6 +438,17 @@ public void removeAttachedFile(WorkflowTask task, String fileInode) throws DotDa
 		return actions;
 	}
 
+	@CloseDBIfOpened
+	public List<WorkflowAction> findActions(List<WorkflowStep> steps, User user) throws DotDataException,
+			DotSecurityException {
+		final ImmutableList.Builder<WorkflowAction> actions = new ImmutableList.Builder<>();
+        for(WorkflowStep step : steps) {
+			actions.addAll(workFlowFactory.findActions(step));
+		}
+		List<WorkflowAction> actionsList = APILocator.getPermissionAPI().filterCollection(actions.build(), PermissionAPI.PERMISSION_USE, true, user);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "actionsList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)